pkg/apis/featurestates/v1alpha1/types.go

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Should the feature states CRD fall under `apis` or `internalapis`?

Reply:
it is internal CRD, user should not be creating this CR, so, do you want to move this at https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/pkg/internalapis 

I see migration CRD is also some internal CRD, the system is creating, do we also want to move that too in internalapis ?
https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/pkg/apis/migration

I am not sure about how folder internalapis will help?

Reply:
Just differentiating CRD's that serve as API's and CRD's that vsphere-csi uses internally. I agree that migration should be moved.  

> it is internal CRD, user should not be creating this CR

Based on this, i think this should be moved too?

Reply:
done.
moved to `pkg/internalapis/featurestates`